src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/DataContractJsonSerializer.cs

```diff
@@ -328,10 +328,12 @@ public void WriteObject(Stream stream, object graph)
 #endif
         }
 
+#if !MERGE_DCJS
```

Comment:
Actually NetNative doesn't use this method since it is calling to the _jsonSerializer which is of type JavascriptSerializer. So this method is not needed in NetNative or when we're using the NetNative DCJS.